mods/sgssb/moves.js

```diff
@@ -568,36 +568,57 @@ exports.BattleMovedex = {
 		target: "self",
 		type: "Water",
 	},
-	// Gligars
-	daredevil: {
+	// Lycanium Z
+	altstorm: {
 		accuracy: 100,
-		basePower: 70,
-		basePowerCallback: function (pokemon, target, move) {
-			if (this.willMove(target)) {
-				this.debug("Power doubled for going first");
-				return move.basePower * 2;
-			}
-			return move.basePower;
-		},
+		basePower: 30,
 		category: "Physical",
 		onPrepareHit: function (target, source) {
 			this.attrLastMove('[still]');
-			this.add('-anim', source, "Giga Impact", target);
+			this.add('-anim', source, "Uproar", target);
 		},
-		desc: "Power doubles if the user goes first.",
-		shortDesc: "Power doubles if the user goes first.",
-		id: "daredevil",
+		desc: "Hits 10 times. Confuses the target after.",
+		id: "altstorm",
 		isNonStandard: true,
-		name: "Daredevil",
+		name: "Alt Storm",
 		pp: 10,
 		priority: 0,
 		flags: {protect: 1, mirror: 1},
-		secondary: false,
+		multihit: 10,
```

Comment:
1 This may not work
2 This is really OP

Reply:
1. It works

2. How about 3/5?

Reply:
that works